topi/include/topi/transform.h

```diff
@@ -365,6 +365,101 @@ inline Array<Tensor> split(const Tensor& x,
   return result;
 }
 
+/*!
+* \brief strided_slice of a tensor
+*
+* \param x The input tensor
+* \param begin The indices to begin with in the slicing
+* \param end Indicies indicating end of the slice
+* \param stride Specifies the stride values, it can be negative
+* in that case, the input tensor will be reversed in that particular axis
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the split operation
+*/
+inline Tensor strided_slice(const Tensor& x,
+                            const Array<Expr> begin,
+                            const Array<Expr> end,
+                            const Array<Expr> stride,
+                            std::string name = "tensor",
+                            std::string tag = kInjective) {
+  size_t src_tensor_dim = static_cast<size_t>(x->shape.size());
+  std::vector<int> begin_val = GetConstIntValues(begin, "begin");
```

Comment:
use int64_t for content value types to support big arrays , when begin's type is int64

Reply:
@tqchen : GetConstIntValues() returns array<int> while it fills the elements with <int64>, should i change the GetConstIntValues() return values? otherwise i need to write a new api which will return array<int64>.
Please suggest.